### DISCUSSION TITLE
r/2023-rv-travel-camping-and-vanlife

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#38] [Announcements]
# r/2023-rv-travel-camping-and-vanlife

2023，使用房车这一年

https://blog.ops-coffee.cn/r/2023-rv-travel-camping-and-vanlife

**Comment by cordream** [comment]
羡慕如此生活

**Comment by 24code** [comment]
请问房车出行有遇到过什么不友好的事情吗。

**Reply by ops-coffee** [reply]
仔细想了想，除了雪花潭遇到的变速箱问题外，出行途中还没遇到过什么不友好的事情，也就是因车的问题有不友好，因人或事还没有

**Comment by varzy** [comment]
好精彩的一年～

**Comment by baozha2003** [comment]
小朋友跟着如何打发时间，本身自己有小孩，也喜欢这样细细品味，但去这些风景地小朋友坐不住。。

**Reply by ops-coffee** [reply]
小孩子确实坐不住，慢慢引导，出去停下里就陪他一起去“探险”，慢慢他就习惯并且喜欢了

**Comment by cuishuang** [comment]
需要停在营地里吗~好像普通停车场不让停房车

**Reply by ops-coffee** [reply]
就普通停车场，没有去过营地

**Comment by fcatat** [comment]
“心有桃花源，处处乌托邦”，非常有感触：）

**Comment by nilevols** [comment]
着实令人羡慕
